pilz_control/CMakeLists.txt

```diff
@@ -24,6 +24,20 @@ catkin_package(
 
 add_definitions(-std=c++11)
 
+# To switch compiler see: https://stackoverflow.com/questions/7031126/switching-between-gcc-and-clang-llvm-using-cmake/12843988#12843988
```

Comment:
Could you auto detect the compiler and only set the respective flags if CLANG is enabled?

Reply:
See PR #281